notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -1138,13 +1135,13 @@
       "source": [
         "### Drift detection during monitoring\n",
         "\n",
-        "The feature input drift detection will occur at the next monitoring interval. In this tutorial, you set the monitoring interval to one hour. So, in about an hour your monitoring job will go from `OFFLINE` to `RUNNING`. While running, it will analyze the logged sampled tables from the predictions during this interval and compare them to the previous monitoring interva distribution.\n",
+        "The feature input drift detection will occur at the next monitoring interval. In this tutorial, you set the monitoring interval to one hour. So, in about an hour your monitoring job goes from **OFFLINE** to **RUNNING**. While running, it analyzes the logged sampled tables from the predictions during this interval and compares them to the previous monitoring interva distribution.\n",
```

Comment:
remove "will" and change "occur" to "occurs":

- The feature input drift detection occurs at the next monitoring interval.

Typo:

- previous monitoring interva distribution.\n",

Should be:

- previous monitoring interval distribution.\n",